include/Utils.h

```diff
@@ -0,0 +1,29 @@
+#ifndef INCLUDE_UTILS_H_
+#define INCLUDE_UTILS_H_
+
+#include <string>
+#include <vector>
+
+#include "BaseTransformation.h"
+
+using clang::tooling::ClangTool, llvm::cl::OptionCategory;
+using std::string, std::vector;
+
+static OptionCategory TransformationCategory("Transformation");
+
+class Runner {
+    /* Class for creating ClangTool form .yaml config specified in argv as a first argument
```

Comment:
I think it's better to parse config outside the runner, so `Runner` accept `vector<ITransformation>`. And `run` accept `string` for the file path.

Reply:
It's better because after that you can import Runner from another module and use it without dumping configs to the disk.